avocado/plugins/list.py

```diff
@@ -146,6 +151,19 @@ def configure(self, parser):
                             'Current: %(default)s')
         loader.add_loader_options(parser)
 
+        filtering = parser.add_argument_group('filtering parameters')
```

Comment:
it'd be nice to create a shared params for `run` and `list` similarly as in `avocado-vt` to avoid this duplicity...